lib/kvdb/kvdb_kvs.h

```diff
@@ -36,6 +37,7 @@ struct kvdb_kvs {
     struct ikvs            *kk_ikvs;
     struct viewset         *kk_viewset;
     struct ikvdb_impl      *kk_parent;
+    enum vcomp_default      kk_vcomp_default;
```

Comment:
I would like to combine all these vcomp members into a since anon struct. Is anyone gonna fight me on it? 🥊 😄

Reply:
Just gonna say not worth it if there are no other changes to this PR